bindata/v3.11.0/kube-apiserver/deployment.yaml

```diff
@@ -84,3 +84,5 @@ spec:
           secretName: serving-cert
```

Comment:
Why is this file called `kube-apiserver/deployment.yaml` when its controller-manager ?

Reply:
> Why is this file called `kube-apiserver/deployment.yaml` when its controller-manager ?

It's content is a deployment

Reply:
`deployment.yaml` seems fine, just that the directory was `kube-apiserver`?
Its an trivial curiosity.